src/View/Helper/FormHelper.php

```diff
@@ -1049,7 +1053,7 @@ public function input($fieldName, array $options = [])
             $options['hiddenField'] = '_split';
         }
 
-        $input = $this->_getInput($fieldName, $options);
+        $input = $this->_getInput($fieldName, $options + ['label' => $optionsLabel]);
```

Comment:
Why not keep the key `optionsLabel` / `labelOptions` here too?  Changing it to `label` is not intuitive.